.github/workflows/build.yml

```diff
@@ -2,9 +2,9 @@ name: build tests
 
 on:
   push:
-    branches: [ chibios-21.6.x ]
+    branches: [ chibios-21.11.x-dev ]
   pull_request:
-    branches: [ chibios-21.6.x ]
+    branches: [ chibios-21.11.x-dev ]
```

Comment:
May have to be changed to just `chibios-21.11.x`. I don't know what to do with this one.

Reply:
This needs to be reverted back for CI to work and validate this PR

Reply:
I've pushed a chibios-21.11.x branch as well, you can use that